### PR TITLE
updates for k8s monitoring

### PR DESCRIPTION
**Pull request by joshuafernandes**
## Pull request checklist

Use the following list to make sure your PR fits the Besu doc quality standard.

### Before creating the pull request

Make sure that:

- [x] [all commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] you read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Contributing+to+documentation).
- [x] you have [tested your changes locally](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewTheDocumentation) before submitting them to the community for review.

### After creating your pull request and tests finished

Make sure that:

- [ ] you fixed all the issues raised by the tests, if any.
- [ ] you verified the rendering of your changes on [ReadTheDocs.org PR preview](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewwithReadTheDocs)
  and updated the testing link (see [Testing](#testing)).

## Describe the change

Updating kubernetes monitoring details with new charts from prometheus-community and elastic.co

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

### For tools changes

- [ ] CircleCI workflow
- [ ] Build and QA tools (lint, vale,…)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] ReadTheDocs configuration
- [ ] GitHub integration

### CODE REVIEW

**Review by alexandratran — APPROVED** [review]
One suggestion, then LGTM

File: docs/Tutorials/Kubernetes/Deploy-Charts.md [hunk 1]
```diff
@@ -75,21 +75,32 @@ which allow Prometheus to scrape metrics from the pod at a specified port and pa
         prometheus.io/path: "/metrics"
 ```
 
-Update the admin `username` and `password` in the [monitoring values file](https://github.com/ConsenSys/quorum-kubernetes/blob/master/dev/helm/values/monitoring.yml),
-then deploy the chart using:
+Update the admin `username` and `password` in the [monitoring values file](https://github.com/ConsenSys/quorum-kubernetes/blob/master/dev/helm/values/monitoring.yml).
+Also remember to configure alerts to the receiver of your choice, that is email, Slack etc and then deploy the chart using:
```
[the remark this answers is not in the document]
Comment (joshuafernandes) [reply docs/Tutorials/Kubernetes/Deploy-Charts.md]: :+1: Thankyou @alexandratran

File: docs/Tutorials/Kubernetes/Deploy-Charts.md [hunk 2]
```diff
@@ -75,21 +75,32 @@ which allow Prometheus to scrape metrics from the pod at a specified port and pa
         prometheus.io/path: "/metrics"
 ```
 
-Update the admin `username` and `password` in the [monitoring values file](https://github.com/ConsenSys/quorum-kubernetes/blob/master/dev/helm/values/monitoring.yml),
-then deploy the chart using:
+Update the admin `username` and `password` in the [monitoring values file](https://github.com/ConsenSys/quorum-kubernetes/blob/master/dev/helm/values/monitoring.yml).
+Configure alerts to the receiver of your choice (for example email or Slack), then deploy the chart using:
```
Comment (alexandratran) [inline docs/Tutorials/Kubernetes/Deploy-Charts.md:79]:
```suggestion
Configure alerts to the receiver of your choice (for example, email or Slack), then deploy the chart using:
```

Comment (joshuafernandes) [reply docs/Tutorials/Kubernetes/Deploy-Charts.md]: :+1:

**Pull request merged**